src/routes/safe/store/actions/utils.js

```diff
@@ -0,0 +1,51 @@
+// @flow
```

Comment:
@fernandomg I moved the common logic for createTransaction and approveTransacton to this file.

Reply:
:pray: thanks!